src/App.js

```diff
@@ -1,42 +1,40 @@
-import React, { useState } from 'react';
-import { Route } from 'react-router-dom';
-import data from './data';
-
+import React, { useState } from "react";
+import { Route } from "react-router-dom";
+import data from "./data";
+import { ProductContext } from "./contexts/ProductContext";
+import { CardContext } from "./contexts/CardContext";
 // Components
-import Navigation from './components/Navigation';
-import Products from './components/Products';
-import ShoppingCart from './components/ShoppingCart';
+import Navigation from "./components/Navigation";
+import Products from "./components/Products";
+import ShoppingCart from "./components/ShoppingCart";
 
 function App() {
-	const [products] = useState(data);
-	const [cart, setCart] = useState([]);
+  const [products] = useState(data);
+  const [cart, setCart] = useState([]);
+
+  const addItem = item => {
+    // add the given item to the cart
+    setCart([ ...cart, item]);
+  };
 
-	const addItem = item => {
-		// add the given item to the cart
-	};
+  const removeItem = id =>{
+	setCart([]);
+  };
```

Comment:
inside your removeItem function, you're passing it an id however, you're not using it anywhere.  Instead, you're just setting the cart to an empty array therefore just removing all items.  Is that the effect you were looking for?